packages/lesswrong/themes/defaultPalette.ts

```diff
@@ -523,5 +523,9 @@ export const defaultComponentPalette = (shades: ThemeShadePalette): ThemeCompone
   },
   warning: {
     main: "#ff9800",
-  }
+  },
+  greenUpvote: {
+    main: "#61CD5A",
+    light: "#70d26a",
+  },
```

Comment:
I feel like this green is too light - it's pretty hard to see, esp in light mode (also I did a quick colorblindness test and this green looks very similar to the grey for the most common form of colorblindness) @agnesstenlund FYI

<img width="676" alt="Screen Shot 2023-09-11 at 4 15 05 PM" src="https://github.com/ForumMagnum/ForumMagnum/assets/9057804/ab021c05-7e4f-46bf-bc34-dc2bdd83437d">